appveyor.yml

```diff
@@ -4,7 +4,7 @@ init:
 # Test against these versions of Node.js.
 environment:
   matrix:
-    - nodejs_version: "13.0"
```

Comment:
I assume this was necessary in order to make the builds pass, meaning that this PR introduces major breaking changes. Which versions of NodeJS will this now work with? This will be important for release notes & change log as many users of this are using NodeJS versions far behind the latest.

Reply:
This PR passes _without_ these changes so I reverted them but I believe that only Node.js 13 (and possibly 12) are compatible with building on Python 3.  @joaocgreis @gengjiawen @rvagg @sam-github please let us know if I am wrong on that.

Reply:
Building on Py3 is still [_experimental_](https://github.com/nodejs/node/blob/master/doc/changelogs/CHANGELOG_V12.md#12.14.0) in Node.js 12.

Reply:
> This PR passes without these changes...but I believe that only Node.js 13 (and possibly 12) are compatible with building on Python 3

OK, so IIUC, this should mean that the 3rd party software can be downloaded & installed on older versions (at least NodeJS v10.x, if not older), but all but the latest versions of NodeJS will be unable to build native modules with them (the main purpose of this package).

BTW, [v12.13.1 release notes](https://github.com/nodejs/node/releases/tag/v12.13.1) mention "Experimental support for building Node.js with Python 3 is improved." so it seems fair to say that 12.x is not fully compatible / ready for Python 3. So if we merge this I think we should only release as beta until this is ironed out and available in LTS.

Reply:
Eventually we need to make it work with Node.js 10, 12 and 13. So I suggested we add all three version.

Reply:
> I suggest we add all three versions

The role of `windows-build-tools` is just to install the software, so its tests only need to cover that, and it seems overkill to run a full test matrix for that rather than just test on the oldest supported version.